lib/src/core/option/extension_transform.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:hmi_core/hmi_core_option.dart';
+import 'package:hmi_core/hmi_core_result.dart';
+
+///
+/// Transforming contained values
+extension Transform<V, E> on Option<V> {
```

Comment:
It is better to rename all extensions to avoid problems when importing. For example, Flutter has a `Transform` widget.

(e.g. `Transform` -> `OptionTransform`, `Extract` -> `OptionExtract`, etc.)